octue/mixins/filterable.py

```diff
@@ -56,7 +56,12 @@ class Filterable:
     def satisfies(self, filter_name, filter_value):
         """ Check that the instance satisfies the given filter for the given filter value. """
         attribute_name, filter_action = self._split_filter_name(filter_name)
-        attribute = getattr(self, attribute_name)
+
+        try:
+            attribute = getattr(self, attribute_name)
+        except AttributeError:
+            raise exceptions.InvalidInputException(f"An attribute named {attribute_name!r} does not exist on {self!r}.")
```

Comment:
Just check this also inherits from AttributeError. If it doesn't, create something like exceptions.FilterAttributeException that does.

Reply:
Looking back, maybe it should just be an `AttributeError` with a custom message?